src/components/pagination/README.md

```diff
@@ -1 +1,228 @@
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Page 2">2</a>
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Page 3">3</a>
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Page 4">4</a>
+    </li>
+    <li class="rvt-pagination__item is-disabled">
+      <a href="#" class="rvt-flex" tabindex="-1" aria-hidden="true">
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+          <g fill="currentColor">
+            <circle cx="8" cy="8" r="2"></circle>
+            <circle cx="14" cy="8" r="2"></circle>
+            <circle cx="2" cy="8" r="2"></circle>
+          </g>
+        </svg>
+      </a>
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Page 20, last page">20</a>
+    </li>
+  </ul>
+</nav>
+```
+
+#### Advanced state
+
+This examples shows the pagination component once a user has navigated to a set of pages where the first, last, previous, and next page links are available.
+
+```
+<nav role="navigation" aria-label="Advanced pagination">
+  <ul class="rvt-pagination">
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Previous page">Previous</a>
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Page 1">1</a>
+    </li>
+    <li class="rvt-pagination__item is-disabled">
+      <a href="#" class="rvt-flex" tabindex="-1" aria-hidden="true">
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+          <g fill="currentColor">
+            <circle cx="8" cy="8" r="2"></circle>
+            <circle cx="14" cy="8" r="2"></circle>
+            <circle cx="2" cy="8" r="2"></circle>
+          </g>
+        </svg>
+      </a>
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Page 6">6</a>
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Page 6">7</a>
+    </li>
+    <li class="rvt-pagination__item is-active" aria-current="true">
+      <a href="#" aria-label="Page 8, current page">8</a>
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Page 9">9</a>
+    </li>
+    <li class="rvt-pagination__item is-disabled">
+      <a href="#" class="rvt-flex" tabindex="-1" aria-hidden="true">
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16">
+          <g fill="currentColor">
+            <circle cx="8" cy="8" r="2"></circle>
+            <circle cx="14" cy="8" r="2"></circle>
+            <circle cx="2" cy="8" r="2"></circle>
+          </g>
+        </svg>
+      </a>
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Page 20, last page">20</a>
+    </li>
+    <li class="rvt-pagination__item">
+      <a href="#" aria-label="Next set of pages">Next</a>
+    </li>
+  </ul>
+</nav>
+```
```

Comment:
I think this is fine for now, but we may want to consider standardizing how we structure the `## Adding the markup` sections. I feel like we have some where we just list the default under the section title, and then add a subsection for Additional options, and sometimes we don't.

Reply:
Good call. I've made a note to re-evaluate these docs for consistent structure once we've worked through them all.